<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountListPage</class>
 <widget class="QWidget" name="AccountListPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>302</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="onlineTab">
      <attribute name="title">
       <string>Online</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="welcomeLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome! If you're new here, you can click the &amp;quot;Add&amp;quot; button to add your Mojang or Minecraft account.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTreeView" name="listView"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="manageAcctsBtnBox">
           <item>
            <widget class="QPushButton" name="addAccountBtn">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rmAccountBtn">
             <property name="text">
              <string>&amp;Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="buttonSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="setDefaultBtn">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the currently selected account as the active account. The active account is the account that is used to log in (unless it is overridden in an instance-specific setting).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>&amp;Set Default</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="noDefaultBtn">
             <property name="toolTip">
              <string>Set no default account. This will cause MultiMC to prompt you to select an account every time you launch an instance that doesn't have its own default set.</string>
             </property>
             <property name="text">
              <string>&amp;No Default</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="uploadSkinBtn">
             <property name="toolTip">
              <string>Opens a dialog to select and upload a skin image to the selected account.</string>
             </property>
             <property name="text">
              <string>&amp;Upload Skin</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="offlineTab">
      <attribute name="title">
       <string>Offline</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout22">
       <item>
        <widget class="QGroupBox" name="offlineBox">
         <property name="title">
          <string>Offline mode name</string>
         </property>
         <layout class="QGridLayout" name="foldersBoxLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="useSelectedNameBtn">
            <property name="text">
             <string>&amp;Use selected account name</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">offlineButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="useFixedNameBtn">
            <property name="text">
             <string>Always use &amp;this offline name:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">offlineButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="mainOfflineNameEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rememberNamesForAccountsBtn">
            <property name="text">
             <string>Remember offline &amp;names per account</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">offlineButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rememberNamesForInstancesBtn">
            <property name="text">
             <string>Remember offline &amp;names per instance</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">offlineButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="offlineButtonGroup"/>
 </buttongroups>
</ui>
